css/style.css

```diff
@@ -1,9 +1,336 @@
+  margin-top: 00.5rem;
+}
+
+/* details h1 */
+.details h1 {
+  text-align: center;
+  font-weight: lighter;
+  font-size: 2.3rem;
+  margin-top: 8.5rem;
+}
+
+/* article */
+.details img {
+  width: 12rem;
+  height: 8rem;
+}
+
+/* article-container */
+.article-container {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  font-size: 1.45rem;
+  margin-top: 3rem;
+  padding: 2rem 23rem;
+}
+
+/* get-karma */
+.get-karma {
+  display: flex;
+  flex-direction: row;
+  margin: 123px 0px;
+}
+
+/* feature-img */
+.feature-img {
+  width: 599px;
+  height: 439px;
+}
+
+/* rect-get-karma */
+.rect-get-karma {
+  background-color: rgba(224, 99, 58, 0.1);
+  width: 841px;
+  height: 439px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+/* rect-get-karma p */
+.rect-get-karma p {
+  font-size: 42px;
+  font-style: italic;
+  font-weight: 400;
+  width: 548px;
+  height: 141px;
+  text-align: center;
+}
+
+/* button-get-karma */
+.button-get-karma {
+  width: 264px;
+  height: 70px;
+  background: #e0633a;
+  border-radius: 8px;
+  color: rgb(255, 255, 255);
+}
+
+/* CSS for store.html */
+
+/* store main section */
+#store-section {
+  display: flex;
+  flex-direction: row;
+}
+
+/* section p */
+form p {
+  font-size: 50px;
+  color: #de6e49;
+  width: 521px;
+  height: 118px;
+  margin-top: 170px;
+}
+
+/* store-section-img */
+.store-section-img {
+  width: 50%;
+}
+
+/* form-container */
+.form-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 50%;
+}
+
+/* input */
+input[type="text"],
+select {
+  height: 53px;
+}
+
+/* fieldset remove border */
+fieldset {
+  border: none;
+}
+
+/* name */
+.name {
+  display: flex;
+  justify-content: space-between;
+  width: 500px;
+  margin-top: 40px;
+}
+
+/* name-group */
+.name-group {
+  display: inline-block;
+}
+/* name-group input */
+.name-group input {
+  display: block;
+  width: 230px;
+}
+
+/* address */
+.address {
+  margin-top: 40px;
+}
+
+/* address input */
+.address input {
+  display: block;
+  width: 500px;
+}
+
+/* address2 */
+.address2 {
+  margin-top: 40px;
+}
+
+/* city */
+.city {
+  display: flex;
+  justify-content: space-between;
+  width: 500px;
+  margin-top: 40px;
+}
+
+/* city-group */
+.city-group {
+  display: inline-block;
+}
+
+/* city-group select */
+.city-group select {
+  display: block;
+  width: 300px;
+}
+
+/* city-group input */
+.city-group input {
+  display: block;
+  width: 130px;
+}
+
+/* select-color */
+.select-color {
+  width: 500px;
+  margin-top: 40px;
+}
+
+/* select-color input */
+.select-color input {
+  margin-right: 15px;
+}
+
+/* select-color label */
+.select-color label {
+  margin-right: 25px;
+}
+
+/* checkbox */
+.checkbox {
+  margin-top: 60px;
+}
+
+/* button-container */
+#button-container {
+  width: 500px;
+}
+
+/* button place my order */
+.button-place-my-order {
+  width: 269px;
+  height: 70px;
```

Comment:
why did you use background color with  background: #e0633a but you use rgb for the color .